src/ably.js

```diff
@@ -74,6 +74,16 @@ class AblyUpdates {
                     })
                 })
 
+                channel.subscribe('OVERFLOW', (message) => {
+                    const { data: payload } = message;
+                    const { event_id, overflow_url } = payload;
```

Comment:
this is not correct is the same payload as data update
plus a  new **params** dictionary where it lives the new attribute
**overflow_url**